spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,10 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      issuer = 'http://localhost:3000'
+      user = sign_in_and_2fa_user
+      loa3_sp_session(issuer: issuer)
```

Comment:
Could you explain the reason for modifying the `loa3_sp_session` method? It looks like you're using the same issuer it was using before. If the reason was to set a `request_url` in the session in order to test that when you click "continue" on the completions page it takes you back to the SP, may I suggest that we don't need that test since the completions page and controller are already tested? The main thing we're concerned with here is to make sure that USPS confirmation takes the user to the completions page.